tests/foreman/cli/test_errata.py

```diff
@@ -459,6 +460,36 @@ def test_negative_install_by_hc_name_without_org_info(self):
             )
         self.assertIn('Error: Could not find organization', context.exception.stderr)
 
+    @tier3
+    @upgrade
+    def test_positive_list_affected_chosts(self):
+        """View a list of affected content hosts for an erratum
+
+        :id: 3b592253-52c0-4165-9a48-ba55287e9ee9
+
+        :Setup: Errata synced on satellite server.
+
+        :Steps: host list --search "applicable_errata = <erratum_id>"
+            --organization-id=<org_id>
+
+        :expectedresults: List of affected content hosts for an erratum is
+            displayed.
+
+        :CaseAutomation: automated
+
+        """
+        with ssh.get_connection() as connection:
+            result = connection.run(
+                'hammer host list --search "applicable_errata = {}" --organization-id={}'
+                ' --fields="Name"'.format(FAKE_2_ERRATA_ID, self.org['id'])
+            )
+            for virtual_machine in self.virtual_machines:
+                match = re.search(r'{}'.format(virtual_machine.hostname), str(result.stdout))
```

Comment:
Can we at least add `\b` around `{}`? So we know we didn't match hostname that happens to be substring of some longer thing?

Not sure how reasonable it is to expect something like that to happen, but matching substring is one of the most common mistakes in regexp I've seen. Better to be safe than sorry.

Reply:
OK